src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -62,7 +62,8 @@ public void execute_invalidCommandFormat_throwsParseException() {
     @Test
     public void execute_commandExecutionError_throwsCommandException() {
         String deleteCommand = "delete 9";
-        assertCommandException(deleteCommand, MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        String expectedString = String.format(MESSAGE_EMPTY_PERSON_LIST, "delete");
+        assertCommandException(deleteCommand, expectedString);
```

Comment:
@beginner-web-developer just to double check this is testing for an empty list right

Reply:
I think so